docs/en/integrations/data-ingestion/data-formats/templates-regex.md

```diff
@@ -0,0 +1,244 @@
+---
+sidebar_label: Regexp and templates
+sidebar_position: 3
+slug: /en/integrations/data-formats/templates-regexp
+---
+
+# Importing and exporting custom text data using Templates and Regex in ClickHouse
```

Comment:
Maybe also add Markdown output format? This format can be useful (not sure it this section is the best for this format)

Reply:
Sounds reasonable, handy format for sharing on Github.
Added PR for markdown docs: https://github.com/ClickHouse/ClickHouse/pull/45651
After that's pushed -> I'll add a mention in the guide.